# HG changeset patch
# User Avinoam Kalma <a.kalma@gmail.com>
# Date 1445291520 25200
#      Mon Oct 19 14:52:00 2015 -0700
# Branch stable
# Node ID 751b9cc78904faae5490cf61d0763be82f773c59
# Parent  410b1b3d109e4acf02a10cae3e64eb58748c2d98
Fix issue with gnuplot, subplot, and 2 y-axes (bug #45822).

* __go_draw_axes__.m: Fix typo and unset 'y2tics' as well as 'x2tics'.

diff --git a/scripts/plot/util/private/__go_draw_axes__.m b/scripts/plot/util/private/__go_draw_axes__.m
--- a/scripts/plot/util/private/__go_draw_axes__.m
+++ b/scripts/plot/util/private/__go_draw_axes__.m
@@ -141,17 +141,17 @@ function __go_draw_axes__ (h, plot_strea
   ## Reset all labels, axis-labels, tick-labels, and title
   ## FIXME: We should have an function to initialize the axis.
   ##        Presently, this is dispersed in this function.
   fputs (plot_stream, "unset label;\n");
   fputs (plot_stream, "unset xtics;\n");
   fputs (plot_stream, "unset ytics;\n");
   fputs (plot_stream, "unset ztics;\n");
   fputs (plot_stream, "unset x2tics;\n");
-  fputs (plot_stream, "unset x2tics;\n");
+  fputs (plot_stream, "unset y2tics;\n");
 
   if (! isempty (axis_obj.title))
     t = get (axis_obj.title);
     if (isempty (t.string))
       fputs (plot_stream, "unset title;\n");
     else
       colorspec = get_text_colorspec (t.color, mono);
       [tt, f, s] = __maybe_munge_text__ (enhanced, t, "string");
